NEWS.txt

```diff
@@ -3,182 +3,64 @@ Release Notes
 =============
 
 
-Theano 0.10.0beta1 (9th of August, 2017)
-========================================
+Theano 0.10.0beta2 (25th of August, 2017)
```

Comment:
Maybe we want to update this?

Reply:
Indeed, I will also have to rebase this PR when we are ready to make the beta.

Which new date ?